src/styles/main.scss

```
@@ -1,3 +1,226 @@
+
 body {
+  font-family: "Roboto", sans-serif;
+
+}
+
+.header {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  
+}
+
+.container {
+  display: flex;
+}
+
+.nav {
+  display: flex;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  align-self: center;
+  margin: 10px;
+}
```

center with flex on header

```suggestion
.logo {
  width: 40px;
  height: 40px;
}
```